Lib/test/test_logging.py

```diff
@@ -4390,6 +4390,15 @@ def rec():
         self.assertNotIn("Cannot recover from stack overflow.", err)
         self.assertEqual(rc, 1)
 
+    def test_get_level_names_mapping(self):
+        def test_get_level_names_mapping(self):
```

Comment:
I don't think you meant this double level of indentation! That causes the test to do nothing effective :disappointed:

Reply:
Whoops, somehow doubling made it hard to spot for me..